libs/libarchfpga/src/physical_types_util.h

```diff
@@ -159,12 +159,15 @@ int get_physical_pin_from_capacity_location(t_physical_tile_type_ptr physical_ti
  */
 std::pair<int, int> get_capacity_location_from_physical_pin(t_physical_tile_type_ptr physical_tile, int pin);
 
+std::string get_class_block_name(t_physical_tile_type_ptr type, int class_physical_num);
```

Comment:
In this file, you need to add a comment to each function (or newly added function) to explain the purpose of the function, its inputs and its output(s).